components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/impl/ActionExecutorServiceImpl.java

```diff
@@ -78,6 +82,8 @@ public class ActionExecutorServiceImpl implements ActionExecutorService {
     private static final ActionExecutionDiagnosticLogger DIAGNOSTIC_LOGGER = new ActionExecutionDiagnosticLogger();
     private final APIClient apiClient;
     private final ExecutorService executorService = ThreadLocalAwareExecutors.newFixedThreadPool(THREAD_POOL_SIZE);
+    private static final String SERVER_ERROR_CODE = "server_error";
```

Comment:
let's not user this underscore. we haven't followed such practice any where

Reply:
And IMO this constant is not needed if you throw the right exception when execution fails

Reply:
This is fixed with 5666dff8e3f964738ed9822a9b553e1cea888e46